css-responsive/css-responsive_responsive-website/css/styles.css

```diff
@@ -244,10 +244,32 @@ width, etc. */
 
 /* Media Query Tablet */
 /* --- */
-/* Replace this comment with your media query. */
+@media screen and (min-width: 37.5rem) {
+  .card-section {
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+  }
+
+  .intro-text,
+  .product-card {
+    width: 45%;
+  }
+
+  .intro-text {
+    align-self: flex-start;
+  }
+}
 /* --- */
 
 /* Media Query Desktop */
 /* --- */
-/* Replace this comment with your media query. */
+@media screen and (min-width: 56.25rem) {
+  .card-section {
```

Comment:
HM, interesting. I googled this type of selector, but it seems like it is inccorect syntax. However, when I preview the website it works as intended. Let's talk on Monday, so you can explain it to me )

Reply:
Let's talk about it :)